lib/shim/message-shim/subscribe-consume.js

```diff
@@ -238,7 +238,6 @@ function createConsumerWrapper({ shim, spec, consumer }) {
      * finalizes transaction name and ends transaction
      */
     function endTransaction() {
-      tx.finalizeName(null) // Use existing partial name.
```

Comment:
i can remove this but realized this was redundant as transaction.end does this as well